### ISSUE TITLE
Playoff/SB games?

### ISSUE THREAD

**Issue by kcmillersean** [ryurko/nflscrapR-data#7]
Does the NFL have the same structure for Playoff and/or Super Bowl games? Would be really cool to analyze how/if teams change their tendencies in the playoffs.

This data is awesome, btw.

**Comment by ryurko** [comment]
Yep, I'll make a folder with all the playoff games since 2009 soon. You can grab the Game IDs using extracting_gameids(year, playoffs = TRUE) from the nflscrapR package. Then just use the game_play_by_play() function on each of those games and should get the playoffs, but I'll make a folder of all of them hopefully this week.

Thanks!
